pyrk/neutronics.py

```diff
@@ -7,7 +7,9 @@
 from reactivity_insertion import ReactivityInsertion
 from timer import Timer
 
+
 class Neutronics(object):
+
```

Comment:
good catch, better for PEP8 compliance. Thanks for adding this all over the place. I had forgotten about that part of pep8. I'm not sure why pythonmode never caught it though.